settings/LoanPolicy/LoanPolicyForm.js

```diff
@@ -36,6 +36,7 @@ import {
   PanelTitle,
   Metadata,
   DeleteEntry,
+  RequestManagementSection,
 } from './components';
```

Comment:
Please make single line import in row 6 `import { getFormValues } from 'redux-form';`